spec.md

```diff
@@ -393,15 +393,6 @@ company). Additionally, some Platforms might modify the service names before
 presenting them to their users. This specification places no requirements on
 how Platforms might expose these values to their users.
 
-##### Dashboard Client Object
```

Comment:
If its just deprecated, I'm not sure we can fully remove it from the spec yet - don't we usually keep the old fields around?

Reply:
Since we're still linking to it elsewhere, I'd be keen to remove this to keep things tidier. You can still find out what it is in the versioned profile.md - it's just moved. What do you think?

Reply:
By that logic we should remove all CF-isms from the core spec **today**.  :-)   I think each version of a spec should be complete on its own. It feels wrong to mention something that is only defined in an old version of the spec.